Models/NHES/Nuclear/CoreSubchannels/Regions_3_CHF_EPRI_inside_FC_newtry.bak-mo

```diff
@@ -0,0 +1,805 @@
+within NHES.Nuclear.CoreSubchannels;
```

Comment:
This is a .bak-mo file without a .mo file. Do we need to change the file extension here and make sure it's loaded for proper runs?

Reply:
Taken out.